group_vars/fender_cluster/vars.yml

```diff
@@ -224,14 +224,14 @@ regular_users:
 #
 pfs_mounts: [
   { pfs: 'ecst01',
```

Comment:
Overlooked this one in the first review. This key needs to match the one used in the lfs_mounts variable, so also needs to be changed back to ecst02.